### PR TITLE
Added initial fine-tuned CNNs on base Fruits-360

### PR DESCRIPTION
**Pull request by ShivanshSrivastava1**
These notebooks download the vanilla Fruits-360 dataset using the Kaggle API to fine-tune the pre-trained ResNet18, VGG16, ViT, and DeiT models. Based on the results, DeiT proved to be the most stable CNN for our ingredient detection.

### CODE REVIEW

**Comment by anuvabanwasi** [comment]
Well-formatted and structured code. Good job with clear train function. Especially liked how you printed the net so it shows the architecture of each model. Also training loss and validation loss looks good. Out of the 4, DeiT shows particularly strong results. Interesting that ViT also did pretty well.
